action/HybridPlanner.action

```diff
@@ -1,5 +1,9 @@
+# This is a modified version of GlobalPlanner.action from ros-planning/moveit_msgs.
+# A JointTrajectory was added so that timing information is retained.
+
 # Motion planning request to the hybrid planning architecture
 string planning_group
+trajectory_msgs/JointTrajectory trajectory
 moveit_msgs/MotionSequenceRequest motion_sequence
```

Comment:
Same comment about MotionSequenceRequest containing a JointTrajectory already, if you dig deep enough.

Reply:
resolving since convo regarding this is in an above thread